src/celeritas/global/alongstep/detail/UrbanMsc.hh

```diff
@@ -0,0 +1,177 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/global/alongstep/detail/UrbanMsc.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Assert.hh"
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "celeritas/Types.hh"
+#include "celeritas/em/data/UrbanMscData.hh"
+#include "celeritas/em/distribution/UrbanMscScatter.hh"
+#include "celeritas/em/distribution/UrbanMscStepLimit.hh"
+#include "celeritas/global/alongstep/AlongStep.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Apply Urban multiple scattering to a track.
+ */
+class UrbanMsc
```

Comment:
Wasn't this defined in [the another file](https://github.com/celeritas-project/celeritas/pull/457/files#diff-4828f3e911e6f130b9fa8a108f319437347357861477cf84ceb65825cd6d4a66) or is this duplication under detail namespace intended for different uses?

Reply:
Yep the other one was a zombie file :(